src/kvirc/ui/KviConsoleWindow.cpp

```diff
@@ -492,7 +492,7 @@ void KviConsoleWindow::closeEvent(QCloseEvent *e)
 			if(!KVI_OPTION_BOOL(KviOption_boolAlwaysDisconnectClosingConnectedConsole))
 			{
 				switch(QMessageBox::warning(this,
-					__tr2qs("Confirmation - KVIrc"),
+					__tr2qs("Confirm Connections Close - KVIrc"),
```

Comment:
Why not make this one consistent with the one below it? (L530). IMO `Confirm Close` sounds cleaner since `Connections` isn't possible as the context is only maintaining a single connection.